src/resources/js/tickets-registration-page.js

```diff
@@ -526,6 +578,68 @@ console.log(value);
 		}
 	);
 
+	/**
+	 * Handle AR submission.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	obj.document.on(
+		'click',
+		obj.selector.checkoutButton,
+		function( e ) {
+			e.preventDefault();
+			var $button      = $( this );
+			var $metaForm    = $( obj.selector.metaForm );
+			var $errorNotice = $( '.tribe-tickets-notice--error' );
+			var isValidForm  = obj.validateForm( $metaForm );
+
+			if ( ! isValidForm[ 0 ] ) {
+				$([document.documentElement, document.body]).animate(
+					{ scrollTop: $( '.tribe-tickets__registration' ).offset().top },
+					'slow'
+				);
+
+
+				$( '.tribe-tickets-notice--error__count' ).text( isValidForm[ 1 ] );
+				$errorNotice.show();
+
+				return false;
+			}
+
+			$errorNotice.hide();
+
+			// save meta
+			var params = {
+				provider: obj.providerId,
+				tickets : {},
+				meta    : obj.getMetaForSave(),
+				post_id : obj.postId,
+			};
+
+			$.ajax({
+				type: 'POST',
+				url: '/wp-json/tribe/tickets/v1/cart',
```

Comment:
Be aware that REST API prefixes can be customized, we should leverage something to account for this.